### PR TITLE
Remove shim.php to set legacy class aliases

### PR DESCRIPTION
**Pull request by zebraf1**
Legacy class aliases are no longer needed in PHPUnit 6.5.0 or later, updated composer requirements.

PHPUnit 6.5.0 is the first version to require phpunit/phpunit-mock-objects: ^5.0
https://packagist.org/packages/phpunit/phpunit#6.5.0

phpunit/phpunit-mock-objects 5.0 is the first version where Mock classes were namespaced:
https://github.com/sebastianbergmann/phpunit-mock-objects/commit/b98a0791b426861a9055c91615b19627716c8950

Since PHPUnit 7.2.0 phpunit/phpunit-mock-objects was merged back into the main repository, so it should work with any version since 6.5 (up to current 7.3.5)

Apparently this bumps PHP version requirement to 7.0 which is to be expected at some point.
Perhaps there should be a codeception/stub 3.0 version for PHP7 and keep PHP5 support for 2.* versions?

Fixes https://github.com/Codeception/Codeception/issues/5508

### CODE REVIEW

File: src/shim.php [hunk 1]
```diff
@@ -1,40 +0,0 @@
-<?php
-if (!class_exists('PHPUnit\Framework\TestCase') && class_exists('PHPUnit_Framework_TestCase')) {
-    class_alias('PHPUnit_Framework_TestCase', 'PHPUnit'.'\Framework\TestCase');
-}
-
-if (!class_exists('PHPUnit\Runner\Version')) {
-    class_alias('PHPUnit_Runner_Version', 'PHPUnit\Runner\Version');
-}
-if (class_exists('PHPUnit_Framework_MockObject_Generator')) {
```
Comment (zebraf1) [inline src/shim.php]: After codeception and phpunit upgrade I started receiving an error while using php-vcr.
PHPUnit_Framework_MockObject_Generator class does not exist for me but it was still trying to autoload it and raised a runtime error:
[PHPUnit\Framework\Exception]
include(PHPUnit_Framework_MockObject_Generator.php): failed to open stream:
"VCR\Util\StreamProcessor::stream_open" call failed

stream_open function was trying to find PHPUnit_Framework_MockObject_Generator.php and returned false.

I am not sure why this error occurred only when running codeception tests with coverage

**Comment by Naktibalda** [comment]
@zebraf1 I apologize for a very late response.
If you could update this branch (rebasing it on `Codeception:master` would be the best) to resolve conflicts, I would release 3.0.0.

Direct dependency on phpunit was replaced with dependency on `"codeception/phpunit-wrapper": ">6.0.15 <6.1.0 | ^6.6.1 | ^7.7.1 | ^8.0.3"`
To drop support for PHPUnit 5 and PHPUnit 6 < 6.5 change it to `"^6.6.1 | ^7.7.1 | ^8.0.3"`

**Comment by zebraf1** [comment]
Merged with master and resolved conflicts

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-  - 5.6
   - 7
   - 7.1
   - 7.2
```
Comment (Naktibalda) [inline .travis.yml]: For some reason PHP 7.3 is missing here.

**Comment by Naktibalda** [comment]
I cleaned up your pull request and raised a new PR at #17

**Comment by zebraf1** [comment]
Thanks!

**Pull request closed without merging**